_im-plugin/ism/policies.md

```diff
@@ -187,18 +187,20 @@ Allows you to reduce the number of primary shards in your indexes. With this act
         "my-alias": {}
       }
     ],
+    "switch_aliases": true,
     "force_unsafe": false
 }
 ```
 
 Parameter | Description | Type | Example | Required
 :--- | :--- |:--- |:--- |
-`num_new_shards` | The maximum number of primary shards in the shrunken index. | integer | `5` | Yes, however it cannot be used with `max_shard_size` or `percentage_of_source_shards`
-`max_shard_size` | The maximum size in bytes of a shard for the target index. | keyword | `5gb` | Yes, however it cannot be used with `num_new_shards` or `percentage_of_source_shards`
+`num_new_shards` | The maximum number of primary shards in the shrunken index. | Integer | `5` | Yes, however it cannot be used with `max_shard_size` or `percentage_of_source_shards`
+`max_shard_size` | The maximum size in bytes of a shard for the target index. | Keyword | `5gb` | Yes, however it cannot be used with `num_new_shards` or `percentage_of_source_shards`
 `percentage_of_source_shards` | Percentage of the number of original primary shards to shrink. This parameter indicates the minimum percentage to use when shrinking the number of primary shards. Must be between 0.0 and 1.0, exclusive.  | Percentage | `0.5` | Yes, however it cannot be used with `max_shard_size` or `num_new_shards`
-`target_index_name_template` | The name of the shrunken index. Accepts strings and the Mustache variables `{{ctx.index}}` and `{{ctx.indexUuid}}`. | `string` or Mustache template | `{"source": "{{ctx.index}}_shrunken"}` | No
-`aliases` | Aliases to add to the new index. | object | `myalias` | No, but must be an array of alias objects
-`force_unsafe` | If true, executes the shrink action even if there are no replicas. | boolean | `false` | No
+`target_index_name_template` | The name of the shrunken index. Accepts strings and the Mustache variables `{{ctx.index}}` and `{{ctx.indexUuid}}`. | String or Mustache template | `{"source": "{{ctx.index}}_shrunken"}` | No
+`aliases` | Aliases to add to the new index. | Object | `myalias` | No, but must be an array of alias objects
+`switch_aliases` | If `true`, copies the aliases from the source index to the target index. If there is a name conflict with an alias from the `aliases` field, the alias from the `aliases` field is used. | Boolean | `true` | No. The default implicit value is `false`, which means no aliases are copied by default.
```

Comment:
Rewrote for clarity.
```suggestion
`switch_aliases` | If `true`, copies the aliases from the source index to the target index. If there is a name conflict with an alias from the `aliases` field, the alias in the `aliases` field is used instead of the name. | Boolean | `true` | No. The default implicit value is `false`, which means no aliases are copied by default.
```